package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
+    "postinstall": "cp -r ./projects/fsastorefrontstyles/fonts ./src/assets",
```

Comment:
After moving process of copying fonts to schematics, we would need to execute `- yarn ng add @spartacus/fsa-schematics` command during installation. As that change is not released to npm, we need post-install scripts until fsa spartacus release 3.0.0